CheckDrive.Web/CheckDrive.Web/Views/DoctorReviews/Index.cshtml

```diff
@@ -5,13 +5,11 @@
     Layout = "~/Views/Shared/_Layout.cshtml";
 }
 
-<h1>Shifokorlar ko'riklari</h1>
-
 <form asp-controller="DoctorReviews" asp-action="Index">
```

Comment:
Is necessary to be here the **form**? 
Do you think an error will occur if you remove the form?

Reply:
If I remove form, search stops working.